### PR TITLE
data:tr-sangale

### PR DESCRIPTION
**Pull request by tr-sangale**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4176"></a>

### CODE REVIEW

File: data/tr-sangale.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Final year graduate, DevOps and Cloud enthusiast, open source ",
+  "avatar": "https://github.com/tr-sangale.png",
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/tejassangale2",
+      "icon": "FaTwitter"
+    }
```
Comment (Tejender1521) [inline data/tr-sangale.json:12]:
```suggestion
    },
```

File: data/tr-sangale.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://twitter.com/tr-sangale",
+      "icon": "FaGithub"
+    }
```
Comment (Tejender1521) [inline data/tr-sangale.json:17]:
```suggestion
    },
```

File: data/tr-sangale.json [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    {
+      "name": "Follow me on LinkedIn",
+      "url": "https://www.linkedin.com/in/tejas-sangale-9b735b1a6/",
+      "icon": "FaLinkedin"
+    }
```
Comment (Tejender1521) [inline data/tr-sangale.json:22]:
```suggestion
    },
```

**Review by Krishnapro — CHANGES_REQUESTED** [review]
Hey @tr-sangale under the link section, you have to add `,` after the end of social link `}` 

like below example 
```
{
      "name": "Follow me on Twitter",
      "url": "https://twitter.com/tejassangale2",
      "icon": "FaTwitter"
  },
```

Except last one

**Pull request closed without merging**